src/library-authoring/containers/ContainerEditableTitle.tsx

```diff
@@ -8,15 +8,29 @@ import messages from './messages';
 
 interface EditableTitleProps {
   containerId: string;
+  readOnly?: boolean;
   textClassName?: string;
+  // In some cases, the title is already available, but it's retrieved in a list of containers.
+  // In these cases, it's necessary to use this `ContainerEditableTitle` for the optimistic update to work.
+  // By using `placeHolderText`, we can give the illusion that the data has already been loaded before using the real data.
+  placeHolderText?: string;
 }
 
-export const ContainerEditableTitle = ({ containerId, textClassName }: EditableTitleProps) => {
+export const ContainerEditableTitle = ({
+  containerId,
+  readOnly,
+  textClassName,
+  placeHolderText,
+}: EditableTitleProps) => {
   const intl = useIntl();
 
-  const { readOnly, showOnlyPublished } = useLibraryContext();
+  const { readOnly: libReadOnly, showOnlyPublished } = useLibraryContext();
 
-  const { data: container } = useContainer(containerId);
+  if (!readOnly) {
+    readOnly = libReadOnly;
```

Comment:
Instead, should do this [below](https://github.com/open-craft/frontend-app-authoring/pull/90/commits/933f99d49a6ff0a78ab3c7afd355333cd27d68bf#diff-1ae8b7fff8780b37c89ac8669f61267d687fa66f56a6640bf5ef3fe3e4c7f680R62):

```
       readOnly={readOnly || libReadOnly}
```